src/fake_node_modules/powercord/components/Toast.jsx

```diff
@@ -59,7 +59,7 @@ module.exports = class Toast extends React.Component {
 
           <div className='buttons'>
             {state.buttons.filter(b => b).map(button => (
-              <div className='button' onClick={() => button.onClick(this.remove.bind(this))}>
+              <div className='button' id={button.type} onClick={() => button.onClick(this.remove.bind(this))}>
```

Comment:
ew IDs

Also at this pont, why not just using Discord's components instead of copying CSS, for both the whole thing and buttons? You can just use Card and Button to get the same style as Discord's without effort, and this will let themes immediately work on custom Powercord elements (as they're basically Discord elements).

You can then let buttons define "color" and "look", and just throw it in the button component and let Discord apply its own styles. You can then add very little amount of css to finalize the button element, like its width or height.